examples/monorepo-yarn-workspaces/README.md

```diff
@@ -0,0 +1,23 @@
+# yarn workspaces monorepo example
+
+This example setup is using the `yarn workspace` feature in combination with ReScript's `pinned-dependencies`.
+
+**Expected IDE plugin behavior:**
+
+When the editor is opened in one of the subprojects `app` or `common`, the editor extension should correctly detect the `bsc` and `bsb` binary in the monorepo root's `node_modules` directory.
```

Comment:
@ryyppy I'm not sure about this. If one sub-repo in the monerepo has a bs-platform that's on a different version it will likely not end up in the root's `node_modules`?

Reply:
Yes, this was brought up in the original PR and discussion on the Rescript Forum, but if the closest node_modules/bs-latform exists, it will continue to use that one, if not it will start looking higher in the projects hierarchy.

This should have the same result as before but with just better handling of the case where you keep all your node modules in the root of the project.

To achieve a more version specific solution, this would require some work with the bs-config.json.